complete/application/src/test/java/com/example/multimodule/application/DemoApplicationTest.java

```diff
@@ -19,7 +17,7 @@
 
     @Test
     public void contextLoads() {
-        assertThat(myService).isNotNull();
+        assertThat(myService.message() != null);
```

Comment:
Should be `assertThat(myService.message()).isNotNull())`?

Reply:
I tried that (naturally, since that was the original). I get a no such method error for `isNotNull()`.

Reply:
I think you have the wrong import then. Works for me. Maybe your IDE is fubar?

Reply:
I'm not using an IDE. I'm using a text editor and running the build files from the command line. I'll try STS.

Reply:
Same thing in STS.

Reply:
I got it to work, finally, so now it is back to `isNotNull()`.